x-pack/plugins/maps/server/routes.js

```diff
@@ -47,32 +46,70 @@ export function initRoutes(server, licenseUid) {
     }
   });
 
+
+  server.route({
+    method: 'GET',
+    path: `${ROOT}/${EMS_DATA_TMS_PATH}`,
+    handler: async (request, h) => {
+
+      if (!request.query.id ||
+        typeof parseInt(request.query.x, 10) !== 'number' ||
+        typeof parseInt(request.query.y, 10) !== 'number' ||
+        typeof parseInt(request.query.z, 10) !== 'number'
+      ) {
+        server.log('warning', 'Must supply id/x/y/z parameters to retrieve EMS tile');
+        return null;
+      }
+
+      const ems = await getEMSResources(emsClient, mapConfig.includeElasticMapsService, licenseUid, true);
+      const tmsService = ems.tmsServices.find(layer => layer.id === request.query.id);
+      if (!tmsService) {
+        return null;
+      }
+
+      const url = tmsService.url
+        .replace('{x}', request.query.x)
+        .replace('{y}', request.query.y)
+        .replace('{z}', request.query.z);
+
+      const tile = await fetch(url);
```

Comment:
Need to add try/catch here. If the kibana server can not access EMS then the kibana logs are flooded with 

<img width="1371" alt="Screen Shot 2019-05-20 at 3 18 53 PM" src="https://user-images.githubusercontent.com/373691/58052676-b783b280-7b12-11e9-850b-7dc1532cc639.png">

And the client gets 500